ramalama/model.py

```diff
@@ -244,6 +245,25 @@ def run(self, args):
         exec_args = self.build_exec_args_run(args, model_path, prompt)
         self.execute_model(model_path, exec_args, args)
 
+    def perplexity(self, args):
+        self.check_name_and_container(args)
+        model_path = self.get_model_path(args)
+        exec_args = self.build_exec_args_perplexity(args, model_path)
+        self.execute_model(model_path, exec_args, args)
+
+    def build_exec_args_perplexity(self, args, model_path):
+        exec_model_path = MNT_FILE if args.container else model_path
+        exec_args = ["llama-perplexity"]
```

Comment:
Can we do something similar to this for vLLM?

Reply:
Not sure, has to be investigated

Reply:
Well if the perplexity command works for non GGUF models, then we could use the llama.cpp one for both vllm as well as llama.cpp